src/g_knb.c

```diff
@@ -0,0 +1,1030 @@
+/* Copyright (c) 1997-1999 Miller Puckette.
```

Comment:
I don't think this copyright line needs to be here, just the one below.

Reply:
done